processes/utils.py

```diff
@@ -1,6 +1,8 @@
+from dataclasses import dataclass
 from datetime import datetime, timedelta, timezone
 from typing import Optional
 
+
```

Comment:
Just curious, is it the standard to have 2 line breaks for spacing?

Reply:
Yes! 

https://peps.python.org/pep-0008/#:~:text=Surround%20top%2Dlevel%20function%20and,separate%20groups%20of%20related%20functions.